### PR TITLE
lower case the email

### LINKED ISSUE #2057: 🐛 [Bug] Password-Reset/Registration Dialogue is Case-Sensitive
## :bug: Bugreport
When users want to reset their passwords or want to register and they use emails with uppcercase in it, these can not be found and you get an email which says:  _You requested a password reset but unfortunately we couldn't find an account associated with your e-mail address. Did you maybe use another one when you signed up?_

wrong-email

### Steps to reproduce the behavior
1. Open reset Password dialogue
2. Type in your emial with upper case letters

Screenshot - 2019-10-29T171429 064

Screenshot - 2019-10-29T171929 698

### Expected behavior
The given email for reset should be converted to lowercase

### Version & Environment
any

### PR DESCRIPTION
**Pull request by Kachulio1**
## 🍰 Pull request

Sets the user email to lower case to avoid case sensitive mismatch 

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: backend/src/schema/resolvers/passwordReset.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ export default {
   Mutation: {
     requestPasswordReset: async (_parent, { email }, { driver }) => {
       const nonce = uuid().substring(0, 6)
-      return createPasswordReset({ driver, nonce, email })
+      const lowerCasedEmail = email.toLowerCase()
+      return createPasswordReset({ driver, nonce, lowerCasedEmail })
```
Comment (roschaefer) [inline backend/src/schema/resolvers/passwordReset.js]: This is not going to work as `email` and not `lowerCaseEmail` gets desctructured here: https://github.com/Human-Connection/Human-Connection/blob/master/backend/src/schema/resolvers/passwordReset.js#L5

Comment (Kachulio1) [reply backend/src/schema/resolvers/passwordReset.js]: Yep, good catch will fix that.

Comment (roschaefer) [reply backend/src/schema/resolvers/passwordReset.js]: https://github.com/Human-Connection/Human-Connection/pull/2118 :point_left:

**Comment by roschaefer** [comment]
I'm going to close this in favour of #2118 hope you don't mind @Kachulio1 ?

**Pull request closed without merging**